Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```diff
@@ -191,3 +191,24 @@
       - id: RubberStampHos
       - id: SecurityTechFabCircuitboard
       - id: JetpackSecurityFilled
+
+- type: entity
+  id: LockerBlueShieldFilled
+  suffix: Filled
+  parent: LockerBlueShield
+  components:
+  - type: StorageFill
+    contents:
+      - id: WeaponDisabler
+      - id: PinpointerNuclear
```

Comment:
this seems like nuke disk objective validhunter bait. I'm personally not a big fan.

Reply:
I will delete